sdk/src/metrics/instrument_metadata_validator.cc

```diff
@@ -1,6 +1,9 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
+#if OPENTELEMETRY_HAVE_WORKING_REGEX
```

Comment:
The CI is failing for this - you need to include `instrument_metadata_validator.h` before using this macro. The changes look good otherwise.